src/mixins/name-request-mixin.ts

```diff
@@ -63,7 +63,8 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
       !!this.getNrApprovedName(nr) &&
       nr.nrNum &&
       nr.requestTypeCd &&
-      [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION].includes(nr.request_action_cd)
+      [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION, NameRequestTypes.NEW]
```

Comment:
See other comment re: NEW not a valid NR for this; should be CHANGE OF NAME.

Reply:
Confirmed by Linda that a "new" name request cannot be used in a firm correction (and not a "conversion" NR either). Only a "change of name" NR can be used.

Reply:
It is now using "CHG" for firm filings.